lib/public/IUser.php

```diff
@@ -143,6 +143,14 @@ public function canChangePassword();
 	 */
 	public function canChangeDisplayName();
 
+	/**
+	 * check if the backend supports changing email addresses
+	 *
+	 * @return bool
+	 * @since 8.0.0
```

Comment:
Its since 28.0.0? It's a change we are introducing now..

Reply:
IG, current is 27.0.0, so yes from 28.0.0 @akhil1508